src/parser.yy

```diff
@@ -200,7 +201,10 @@ stmt : expr         { $$ = new ast::ExprStatement($1); }
      | var MINUSMINUS  { $$ = new ast::AssignVarStatement($1, new ast::Binop($1, token::MINUS, new ast::Integer(1))); }
      ;
 
-expr : INT             { $$ = new ast::Integer($1); }
+int : MINUS INT    { $$ = new ast::Integer(-1 * $2); }
+    | INT          { $$ = new ast::Integer($1); }
+
```

Comment:
```suggestion
    ;
```
Guess the final semi colon isn't needed then if this is working... still, I think it should be included for consistency with the rest of the rules in this file